### ISSUE TITLE
Questions on non-systemd support

### ISSUE THREAD

**Issue by Dariqq** [linux-surface/iptsd#158]
Hi, 

It looks like the comment on systemd was removed from the README in 4230bdebc0f09c39863af22cc86afc84a3bc0e54. Have there been any changes other than what is said in https://github.com/linux-surface/iptsd/issues/80?

Also could some of the explanations for why systemd is used from the other issue be added to the README (mainly hidraw device changing after suspend) such that they are easier to find?

Additionally would it be possible to document the suboptimal workaround of building with "-Dservice_manager=" and the wrapper script/the equivalent openrc service, which from what I have read should work but requires manual restarting to refind the hidraw device (please correct me if i am wrong with this)? 
Maybe these files could also be added as service_manager options (but clearly indicating that the experience with them is inferior)  to have at least a very basic support..

Thanks.

**Comment by StollD** [comment]
> It looks like the comment on systemd was removed from the README in [4230bde](https://github.com/linux-surface/iptsd/commit/4230bdebc0f09c39863af22cc86afc84a3bc0e54). Have there been any changes other than what is said in #80?

No, i removed it because it isn't really relevant to the project as whole, and we only package it for systemd distributions. 

However ...

> Also could some of the explanations for why systemd is used from the other issue be added to the README (mainly hidraw device changing after suspend) such that they are easier to find?

... I agree that this is useful information, so I am going to add that back in it's own section.

> Additionally would it be possible to document the suboptimal workaround of building with "-Dservice_manager="

That isn't a workaround, thats just how meson works. The option accepts a list of entires, what you posted is just the empty list. I designed it like that, so that we could at some point build a Debian or Arch package that supports multiple service managers.

Right now I agree that it is pretty useless, but changing it to e.g. a boolean would break downstreams and idk if that's worth it.

> the equivalent openrc service, which from what I have read should work but requires manual restarting to refind the hidraw device (please correct me if i am wrong with this)?

Yes, it may break after resuming from suspend.

> Maybe these files could also be added as service_manager options (but clearly indicating that the experience with them is inferior) to have at least a very basic support..

I don't want to ship services that don't work correctly and give the impression that they are supported.
